components/locales/cs-CZ.json

```diff
@@ -235,7 +235,8 @@
         "range": "Množství ${label} musí být mezi ${min}-${max}"
       },
       "pattern": { "mismatch": "${label} se neshoduje se vzorem ${pattern}" }
-    }
+    },
```

Comment:
Missing `Form`

Reply:
This file already had it so I just added the new value to the existing one.

Reply:
That's right, thank you.